module/os/linux/zfs/zpl_file.c

```diff
@@ -989,6 +997,8 @@ static long
 zpl_ioctl(struct file *filp, unsigned int cmd, unsigned long arg)
 {
 	switch (cmd) {
+	case FS_IOC_GETVERSION:
+		return (zpl_ioctl_getversion(filp, (void *)arg));
```

Comment:
It looks like there is a `FS_IOC32_GETVERSION` which should be wired up as well in `zpl_compat_ioctl()`.  The compat code is to handle a 32-bit user space running on a 64-bit kernel.  Not a very common use case these days, but easy enough to add support.